2021-08-31 18:32:59,262 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-08-31 18:32:59,262 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /usr/local/lib/python3.9/site-packages/instabot/bot/bot.py
2021-08-31 18:32:59,263 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-08-31 18:32:59,515 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2021-08-31 18:32:59,737 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2021-08-31 18:32:59,964 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2021-08-31 18:33:00,185 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2021-08-31 18:33:03,097 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [200]>
2021-08-31 18:33:03,097 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'goga_osoka'!
2021-08-31 18:33:03,097 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: True
2021-08-31 18:33:03,311 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [400]>
2021-08-31 18:33:03,312 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/qe/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:03,312 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:03,312 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:03,312 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:33:03,514 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [400]>
2021-08-31 18:33:03,515 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/launcher/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:03,515 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:03,515 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:03,515 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:33:03,729 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: zr/token/result/?device_id=android-d0b0f4c504afc855&token_hash=&custom_device_id=android-d0b0f4c504afc855&fetch_reason=token_expired returned response: <Response [400]>
2021-08-31 18:33:03,729 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/zr/token/result/%253Fdevice_id%253Dandroid-d0b0f4c504afc855%2526token_hash%253D%2526custom_device_id%253Dandroid-d0b0f4c504afc855%2526fetch_reason%253Dtoken_expired","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:03,729 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:03,729 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:03,729 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:33:03,947 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: multiple_accounts/get_account_family/ returned response: <Response [400]>
2021-08-31 18:33:03,948 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/multiple_accounts/get_account_family/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:03,948 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:03,948 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:03,948 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:33:04,171 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [400]>
2021-08-31 18:33:04,171 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/qe/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:04,171 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:04,172 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:04,172 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:33:04,440 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: igtv/browse_feed/?prefetch=1 returned response: <Response [400]>
2021-08-31 18:33:04,441 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/igtv/browse_feed/%253Fprefetch%253D1","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:33:04,441 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:33:04,441 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:33:04,441 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
